pennylane/ops/qubit/qchem_ops.py

```diff
@@ -464,3 +464,146 @@ def _matrix(cls, *params):
     def adjoint(self):
         (theta,) = self.parameters
         return DoubleExcitationMinus(-theta, wires=self.wires)
+
+
+class OrbitalRotation(Operation):
+    r"""OrbitalRotation(phi, wires)
+    Spin-adapted spatial orbital rotation.
+
+    .. figure:: ../../_static/qchem/orbital_rotation_decomposition_extended.png
+        :align: center
+        :width: 100%
+        :target: javascript:void(0);
+
+    .. figure:: ../../_static/qchem/givens_rotation_decomposition.png
+        :align: center
+        :width: 85%
+        :target: javascript:void(0);
+
+    |
+
+    This operation performs a Givens rotation between the orbital bases in the Jordan-Wigner basis using a pair
+    of parallel Givens gates :math:`G(\phi)`, which are equivalent to :class:`~.SingleExcitation()` operation.
+    More precisely, for two neighbouring spatial orbitals :math:`\{|\Phi_{0}\rangle, |\Phi_{1}\rangle\}`,
+    it performs the following transformation
+
+    .. math::
```

Comment:
Really clever way to summarize what the operation is doing! :tada: :brain:

Reply:
I agree!! really nice docstring 🙂 

One suggestion: it would be nice to briefly *describe* the image (in this case, I wasn't originally sure what `G(theta)` referred to, I assume givens rotations?)

Reply:
Actually, now that I look at it again: it might be clearer to remove the single-excitation image, and in the text below, simply say

```rest
:math:`G(\phi)` represents a single-excitation Given's rotation,
implemented in PennyLane as the :class:`~.SingleExcitation` operation.
```
This way a reader can easily click on the link to read more about the `SingleExcitation` operation.

Reply:
I agree. I have added this to the documentation now.

Reply:
Love it!